githubscript/src/githubscript/release_config.py

```diff
@@ -9,13 +9,22 @@
 
 
 def get_release_config(product_config, task_payload, config):
+    # TODO: add unittests for this, including fallback
+    # support repo override for xpi-manifest, but support original workflow for fenix
+    if "allow_github_repo_override" in product_config and product_config["allow_github_repo_override"] is True:
```

Comment:
This could be `if product_config.get("allow_github_repo_override"):` which will be `True` or `False` if it's specified, or `None` if it isn't specified. We'd only enter the block if it were explicitly set to `True`.